libraries/AC_PID/AC_PID_2D.h

```diff
@@ -36,6 +36,7 @@ class AC_PID_2D {
     Vector2f get_i() const;
     Vector2f get_d() const;
     Vector2f get_ff();
+    Vector2f get_error() const {return _error;}
```

Comment:
We can make this a const reference and add in some spaces.

const Vector2f& get_error() const { return _error; }